packages/core/src/GlobalStyles/GlobalStyles.tsx

```diff
@@ -34,6 +34,8 @@ const cssReset = css`
   body {
     text-rendering: optimizeSpeed;
     line-height: 1.5;
+    /* Improves readability of lowercase L (l) and uppercase i (I) */
+    font-feature-settings: 'ss05' on;
```

Comment:
IMO this should go into the other body declaration that sets the font as it depends on the font supporting it.